### PR TITLE
NAS-121257 / 23.10 / F60 support for UI and mock utility

### PR DESCRIPTION
**Pull request by dszidi**
Test requirements: 

- Ensure product image is visible on main dashboard SysInfo widget
- Ensure product image is visible under system/general support card
- Make sure enclosure ui is available and shows visualization for F60

Do the above scenarios against real machine (check ticket for url)
Also try out mock utility...

`yarn ui mock` and select options "included" > "f60.config.json"

### CODE REVIEW

**Comment by darkfiberiru** [comment]
@dszidi As you can see here there are 3 identical models F60/F100/F130. Enclosure is identical for all 3.

File: src/app/core/testing/utils/enclosure-templates/mock-f60.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      Device: 'nvme12n1',
+    },
+    name: 'Array Device Slot',
+    descriptor: 'Disk #1',
+    status: 'OK',
+    value: 'None',
+    value_raw: '0x1000000',
+  };
+
+  // Need to obtain output of real F60 empty slot to verify
```
Comment (darkfiberiru) [inline src/app/core/testing/utils/enclosure-templates/mock-f60.ts:24]: Have you compared now that you have access?

**Review by undsoft — APPROVED** [review]
Looks good, thanks for the quick turnaround.

**Review by denysbutenko — APPROVED** [review]
image

image

image

**Pull request merged**